module/applications/activity/activity-usage-dialog.mjs

```diff
@@ -461,9 +460,9 @@ export default class ActivityUsageDialog extends Dialog5e {
    * Perform any pre-processing of the form data to prepare it for updating.
    * @param {SubmitEvent} event          Triggering submit event.
    * @param {FormDataExtended} formData  Data from the submitted form.
-   * @returns {object}
+   * @returns {Promise<object>}
    */
-  _prepareSubmitData(event, formData) {
+  async _prepareSubmitData(event, formData) {
```

Comment:
Had a need for this method to be `async`, but it seemed safe to make the change given that the two places it was currently called were also `async` contexts.